packages/xo-web/src/xo-app/hub/resource.js

```diff
@@ -83,7 +87,7 @@ export default decorate([
                 namespace,
                 id,
                 version,
-                sr: pool.default_SR,
+                sr: resourceParams.mapPoolsSrs[pool.id],
```

Comment:
Can't this be `undefined`?

Reply:
No, It has never undefined according to my tests. and given my algorithm, it must be different from undefined

Reply:
It can be `undefined` if the user didn't select an SR.

Reply:
These values come from required inputs, so they can never be `undefined`

Reply:
When a user don't select a SR, In the UI the default SR will be chosen because it's the default value of the input, but the SR entry will not be present in the map because the user don't trigger a  `onChange` action.

Reply:
You're right, fixed here --> https://github.com/vatesfr/xen-orchestra/pull/4571/files#diff-17032e40a62de5b80aa40071c01441faR79